queue/lib/txgh-queue/job.rb

```diff
@@ -3,7 +3,8 @@
 
 module TxghQueue
   class Job
-    include TxghServer::Webhooks::Github
+    Github = TxghServer::Webhooks::Github
+    Transifex = TxghServer::Webhooks::Transifex
```

Comment:
Can you explain your strategy here? This will make these constants available under `TxghQueue::Job::Github`, etc, but they're really just an alias for the webhooks, right? Can you `include TxghServer::Webhooks` instead?

*EDIT* I've done some experimenting — turns out that TxghQueue::Job.constants returns `[:Github, :Transifex]` either way, so nevermind.